chidioguejiofor/airtech-api
# #165294349: Increase test coverage to 99%

#### What does this PR achieve?

- adds missing tests to the app
- adds new assertion methods to the assertion helpers
- ensures that all tests are passing on CircleCI
- adds new test config in .coverager

#### Any background context?
- Tests give us confidence everything works the same when adding new features or refactoring our app. Test coverage is thus very important

#### New environmental Variables?
- None

#### Manual Testing Steps
- Fetch to this branch via: `git fetch origin ch-improve-test-coverage-165294349`
- Checkout to the branch via: `git checkout ch-improve-test-coverage-165294349`
-  Run test with `  pytest --cov-report=html --cov=airtech_api --cov-branch`
- Open the htmlcov/index.html file in your browser
- You should see that the coverage has increased to 99%
